src/components/TaskStatusBadge/index.tsx

```diff
@@ -89,17 +90,17 @@ const customBadgeColors = (status: string) => {
   switch (status) {
     case TaskStatus.SetupFailed:
       return {
-        fill: purple.light1,
-        border: purple.light2,
-        text: purple.base,
+        fill: purple.dark2,
+        border: purple.dark3,
+        text: purple.light3,
       };
     case TaskStatus.SystemFailed:
     case TaskStatus.SystemUnresponsive:
     case TaskStatus.SystemTimedOut:
       return {
-        fill: purple.base,
-        border: purple.dark1,
-        text: purple.light1,
+        fill: purple.light2,
+        border: purple.dark3,
+        text: purple.dark2,
```

Comment:
These colors might be backwards; I think setup failed is supposed to be light purple, while system failed should be dark purple